### ISSUE TITLE
Estimating Evidence with MultiNest sampler 

### ISSUE THREAD

**Issue by SuvedhaNaik** [21cmfast/21CMMC#82]
Hello,

I want to use 21CMMC to compare different models against some data sets.
I want to perform a Bayesian model comparison to do so.
Since 21CMMC already has the `MultiNest` sampler,
is there a way to get the Bayesian evidence of the model?

Thanks in advance.

Suvedha

**Comment by steven-murray** [comment]
Yes -- use the multinest sampler and read the output files to get the evidence, just as you normally would for `MultiNest`. I hope that helps!
